go.sum

```diff
@@ -0,0 +1,2 @@
+github.com/soda-cdm/documentation v0.0.0-20230525043849-2b985763d78d h1:3xzxcT2yHfhVJWOvnVstPMU9lx3yzZ24RAojwVvHC6I=
```

Comment:
No need to check in this file

Reply:
Seems like this file is still checked in. 
Since you had it pushed it to the fork already you will have to delete the file and push the changes again.